PowerShellEditorServices.build.ps1

```diff
@@ -72,22 +72,31 @@ function Install-Dotnet {
     Invoke-WebRequest "https://dot.net/v1/dotnet-install.$installScriptExt" -OutFile $installScriptPath
     $env:DOTNET_INSTALL_DIR = "$PSScriptRoot/.dotnet"
 
-    $args = if($Version) {
-        '-Version'
-        $Version
-    } elseif ($Channel) {
-        '-Channel'
-        $Channel
+    $splat = if ($script:IsUnix) {
+        if($Version) {
+            '-Version'
+            $Version
```

Comment:
This is arguably clearer as an array like `@('-Version', $Version)` so it's obvious that you're using array splatting